src/partials/device_selection_header.hbs

```diff
@@ -1,9 +1,48 @@
 {{#ifEq device.type "Android"}}
-  <h1 class="heading">PingOne Authenticator</h1>
+  <h1 class="heading-text centered-text icon-heading">
+    <svg viewBox="0 0 24 24" role="presentation" style="width: 32px; height: 32px;">
+      <path
+              d="M16,18H7V4H16M11.5,22A1.5,1.5 0 0,1 10,20.5A1.5,1.5 0 0,1 11.5,19A1.5,1.5 0 0,1 13,20.5A1.5,1.5 0 0,1 11.5,22M15.5,1H7.5A2.5,2.5 0 0,0 5,3.5V20.5A2.5,2.5 0 0,0 7.5,23H15.5A2.5,2.5 0 0,0 18,20.5V3.5A2.5,2.5 0 0,0 15.5,1Z"
+              style="fill: rgb(61, 69, 77);"
+      />
+    </svg><span>PingOne Authenticator</span></h1>
 {{else ifEq device.type "iPhone"}}
-  <h1 class="heading">PingOne Authenticator</h1>
+  <h1 class="heading icon-heading">
+    <svg viewBox="0 0 24 24" role="presentation" style="width: 32px; height: 32px;">
```

Comment:
should we make the inline style apply to all heading icon-heading classes? Not sure if its common among other templates or not.

Reply:
Good thinking, updated